apps/mobile/src/components/GalleryEditor/SortableRow.tsx

```diff
@@ -247,12 +246,7 @@ export function SortableRow({
       left: 0,
       right: 0,
       zIndex: zIndex,
-      transform: [
-        { translateY: translateY.value ?? 0 },
-        {
-          scale: withSpring(isGestureActive.value ? 1.05 : 1),
-        },
-      ],
```

Comment:
Remove the scale for the **row** here. Kind of weird when hovering the token to drag but the **row ** as triggered too